lib/ui/painting/image_decoder_impeller.cc

```diff
@@ -323,7 +323,8 @@ sk_sp<DlImage> ImageDecoderImpeller::UploadTextureToShared(
   texture_descriptor.storage_mode = impeller::StorageMode::kHostVisible;
   texture_descriptor.format = pixel_format.value();
   texture_descriptor.size = {image_info.width(), image_info.height()};
-  texture_descriptor.mip_count = texture_descriptor.size.MipCount();
+  texture_descriptor.mip_count =
+      create_mips ? texture_descriptor.size.MipCount() : 1;
```

Comment:
For my edification, why isn't the texture desciptor telling us the MipCount is 1 here?

Reply:
I added a code path to disable mipmap creation for certain texture uploads. So even though this texture may be big enough to support 3 miplevels we may not want to create them